repositories/strategies/duplicateStrategy.js

```diff
@@ -0,0 +1,214 @@
+const { omit, head, get, isNil } = require("lodash");
+const { getOrUpdateOrderForPageInsert } = require("./spacedOrderStrategy");
+
+const insertData = async (
+  trx,
+  tableName,
+  data,
+  callback,
+  returning = "*",
+  position
+) => {
+  const returnedData = await trx
+    .table(tableName)
+    .insert(data)
+    .returning(returning)
+    .then(callback);
+
+  if (returnedData.order) {
+    const order = await getOrUpdateOrderForPageInsert(
+      trx,
+      returnedData.SectionId,
+      returnedData.id,
+      position
+    );
+
+    await trx
+      .table(tableName)
+      .update({ order: order })
```

Comment:
This could be `.update({order})`

Reply:
I've seen (and done this myself) a few times.  Maybe we can set up a linting rule to enforce this?

Reply:
Yeah we can add it - I have definitely seen it before. I think airbnb eslint does it by default. But its a tech debt.